src/main/features/claim/views/eligibility/not-eligible.njk

```diff
@@ -20,6 +21,15 @@
           + '<a target="_blank" href="%s">form N1</a> ' + newWindow() + '.',
           { postProcess: 'sprintf', sprintf: [ n1FormUrl ]}) | safe }}
         </p>
+      {% elseif reason === NotEligibleReason.MULTIPLE_CLAIMANTS %}
+        <p>{{ t('You can’t use this service if more than one person or business is making the claim') }}</p>
+        <p>{{ t('Use <a target="_blank" href="%s">form N1 </a>' + newWindow() + ' to make your claim..',
+          { postProcess: 'sprintf', sprintf: [ n1FormUrl ]}) | safe }}</p>
+      {% elseif reason === NotEligibleReason.MULTIPLE_DEFENDANTS %}
+        <p>{{ t('Use <a target="_blank" href="%s">Money Claim Online </a>' + newWindow() + ', if you’re claiming against two people.',
+          { postProcess: 'sprintf', sprintf: [ moneyClaimOnline ]}) | safe }}</p>
+        <p>{{ t('If your claim is against more than two individuals or companies, use <a target="_blank" href="%s">form N1 </a>' + newWindow() + ' to make your claim.',
+          { postProcess: 'sprintf', sprintf: [ n1FormUrl ]}) | safe }}</p>
```

Comment:
on line 38 of this file it also goes to mcol, please update now that it is being used in more than one place to use the variable
REQ

Reply:
Done.